src/Flux/layers.jl

```diff
@@ -54,16 +55,16 @@ function diffeq_adjoint(p,prob,args...;u0=prob.u0,kwargs...)
   adapt(T, solve(_prob,args...;kwargs...))
 end
 
-diffeq_adjoint(p::TrackedVector,prob,args...;u0=prob.u0,kwargs...) =
-  Flux.Tracker.track(diffeq_adjoint, p, u0, prob, args...; kwargs...)
+#= diffeq_adjoint(p::TrackedVector,prob,args...;u0=prob.u0,kwargs...) = =#
+#=   Flux.Tracker.track(diffeq_adjoint, p, u0, prob, args...; kwargs...) =#
 
-@grad function diffeq_adjoint(p,u0,prob,args...;backsolve=true,
+@adjoint function diffeq_adjoint(p,u0,prob,args...;backsolve=true,
```

Comment:
it's this right here

Reply:
Why is this not an issue with Tracker?

```
diffeq_adjoint(p::TrackedVector,prob,args...;u0=prob.u0,kwargs...) =
  Flux.Tracker.track(diffeq_adjoint, p, u0, prob, args...; kwargs...)


@grad function diffeq_adjoint(p,u0,prob,args...;backsolve=true,
                              save_start=true,
                              sensealg=SensitivityAlg(quad=false,backsolve=backsolve),
                              kwargs...)
```

https://github.com/JuliaDiffEq/DiffEqFlux.jl/blob/a8105ee042d1f80538d428f1e50acf421f572cde/src/Flux/layers.jl#L57-L63

Why is this not an issue with Tracker?

Reply:
Ah okay. Do Zygote adjoints not need separate function definitions (i.e. without and with `@adjoint`)? If they do then which argument order is correct? Your suggested change has `p,u0,prob...` but the Tracker (non-`@grad`) has `u0` a keyword argument whose default value is supplied by the `prob`. Which should I go with?

Reply:
I guess add the argument